docs/internals/dev.rst

```diff
@@ -20,7 +20,14 @@ Linux or macOS.  Windows is not currently supported.
 * Python 3.7 dev package;
 * Bison 1.875 or later;
 * Flex 2.5.31 or later;
-* Perl 5.8.3 or later.
+* Perl 5.8.3 or later;
+* Zlib (zlibg1-dev on Ubuntu);
+* Readline dev package;
+* Libuuid dev package.
+
+.. zlib, readline and libuuid are required to build postgres. Should be removed
+   when custom postgres build is no longer needed.
```

Comment:
We'll probably continue having postgres as a submodule even when we have a hosted version etc. It just makes it easier for us pin a specific dev version of postgres; and for many people to build edgedb locally without bothering with installing a new system postgres.